.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - uses: actions/checkout@v3.1.0
       - uses: actions/setup-go@v3.3.1
         with:
-          go-version: '1.17'
+          go-version: '1.18'
```

Comment:
Maybe we could simply use `1.19` here? We have `compatibility-test` to check test against `1.18`.

Reply:
SGTM

Reply:
https://github.com/signalfx/splunk-otel-go/pull/1570/commits/c36d9177d2f6bd60da4746944d4601b5534f324a